committees/green-team/community-garden.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: interior-page
+title: 'Community Garden'
+parent: 'Green Team'
+---
+
+# Rutherford Community Garden
+
+The Rutherford Community Garden is located at [West Erie Ave between Grand and Jackson](https://goo.gl/maps/zdgEhzCxa3bNymBRA).
+
+## About the Garden
+The Borough of Rutherford is known for beautiful trees, welcoming parks, and a strong community with a tradition of volunteerism. The “Borough of Trees” first established the community garden along Erie Ave in 2010, with official grand opening in April 2011. 
+
+Through the hard work of the Green Team and scores of volunteers representing many clubs and organizations, The Rutherford Community Garden continues to be a beautiful and engaging place for all residents. The garden is a model of sustainability and recycling. The original funding for the garden came from Sustainable Jersey as a $25,000 grant from Walmart. Many of the beds and pathways were created using recycled stone from the borough’s sidewalks.  Other paths incorporate wood chips from street trees that had reached the end of their lives and are now protecting young trees in our tree nursery. Water barrels capture rainwater and help with irrigation, thus reducing the need for tap water. When necessary, water is  delivered via water-saving drip irrigation. With fifteen planting areas, two learning areas (one boasting a green roof), a community shed, senior and handicap-accessible beds, a compost area and a tree nursery, the garden has transformed an empty lot into a community asset. 
+
+The mission for the garden is to provide the young, and the young at heart, a place to observe and appreciate nature. Given our fast-paced world, we also worked to make the garden low maintenance and user friendly. It is a beautiful and welcoming place created by residents for the residents. 
+
+## Elements of the Garden
+
+### <a name="compost-pile"></a>Compost Pile
```

Comment:
These named anchors aren't necessary, the headers generated by the markdown conversion get page IDs which can be referenced with the hash. View source to see what they are (they're automatically slugified from the text of the header tag).

For example, the H2 above:
https://www.bogosian.dev:38080/committees/green-team/community-garden/#elements-of-the-garden

Reply:
TIL :) I assume I should remove the anchors and redo the request?